src/BinaryInstaller.php

```diff
@@ -207,9 +218,6 @@ protected function installBinary($binary, $version, $url, $sha, $hashalgo = 'sha
         }
 
         if ('.tar.gz' === substr($url, -7)) {
-            // Remove the .tar file from the cache directory if it was
-            // previously extracted.
-            $fs->remove(substr($cacheDestination, 0, -3));
```

Comment:
Is this removed on purpose? It was added in the previous PR, isn't it?

Reply:
@rabbitlair yes I'm actually already doing it a few lines down already :facepalm: